integrations/friendbuy/HISTORY.md

```diff
@@ -1,3 +1,9 @@
+1.0.8 / 2020-08-24
```

Comment:
This looks like a minor feature. Can we make this 1.1.0?

Reply:
Sure, go ahead 👍